aicspylibczi/CziFile.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, czi_filename: types.FileLike, metafile_out: types.PathLike =
                  verbose: bool = False):
         # Convert to BytesIO (bytestream)
         self._bytes = self.convert_to_buffer(czi_filename)
-        self.czi_filename = None
+        self.czi_filename = czi_filename
```

Comment:
Could you modify the PR to delete the variable `self.czi_filename` meaning just remove this line? Then I'll merge it.